src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,98 @@
+import cn from 'classnames';
+import React from 'react';
+
+import { deleteTodo } from '../../api/todos';
+import { useTodos } from '../../hooks/useTodos';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+const TodoItem: React.FC<Props> = ({ todo }) => {
+  const [selectedTodoId, setSelectedTodoId] = React.useState<number>(0);
```

Comment:
Why do you store the selected todo id inside the todo item? And why do you set it when you actually delete it

Reply:
> Why do you store the selected todo id inside the todo item? And why do you set it when you actually delete it

?